### PR TITLE
Fix image missing when using Base64 content

### PR DESCRIPTION
**Pull request by christiansandberg**
When adding a Base64 encoded image to a non-self contained report, only an empty link is created but no image.

### CODE REVIEW

**Comment by christiansandberg** [comment]
@BeyondEvil could you have a look at this? Should be quite trivial.

**Comment by BeyondEvil** [comment]
Thanks! I'm planning on putting in some time during the weekend to look at all open issues and PRs.

**Comment by ssbarnea** [comment]
Please rebase.  Thanks.

**Comment by christiansandberg** [comment]
@ssbarnea I’ve rebased my branch.

**Comment by dhalperi** [comment]
Darn it, I am the third person to create this pr [#298, after #287] with mildly different tests. @BeyondEvil - any way to review/merge this?

Related to #265.  [now it's cross-linked to the issue and maybe a fourth person won't write it :).]

**Comment by dhalperi** [comment]
cc: @ssbarnea

**Comment by BeyondEvil** [comment]
Sorry for not getting to this sooner @christiansandberg 

tl;dr had spine surgery which led to complications.

Anyway, as soon as I'm satisfied with understanding the issue this is fixing (just want to see it with my own eyes) I'm going to merge this.

@dhalperi Sorry you had to do all that work, but you should've checked existing issues and PRs amirite 😉 What I would like for you to do, once this is merged, is to rebase. What will be left is basically that extra test (which is awesome!) but I also want you to update `CHANGES.rst` and give yourself and Christian credit. 😄For version you can just bump the patch number and for date put `unreleased`.

All good?

**Comment by dhalperi** [comment]
@BeyondEvil - your health takes top priority, I hope you're doing okay now. Glad to see you are able to work again. Everyone's having a different thing happening in this crazy time...

I am happy to support in whatever mode is best! Let me know. (Also, if there's a gitter or slack channel that. might better facilitate realtime communication, I can go there)

**Comment by BeyondEvil** [comment]
I'm jimboslice in the official Selenium slack @dhalperi :)

**Comment by dhalperi** [comment]
@BeyondEvil - the official selenium slack HQ link (from [here](https://seleniumhq.herokuapp.com/)) has expired - maybe someone there should refresh it. The question I came in to ask - will it be possible to make a new release with this fix sometime soon?
